Recolor.php

```diff
@@ -19,6 +21,7 @@ class Recolor {
      * Process an image file
      *
      * @return void
+     * @throws \Exception
```

Comment:
It's always worth including a doc block for everything which happens inside your method, even when you're throwing an exception!